plugins/opl2/fmopl.c

```diff
@@ -653,29 +653,28 @@ static void init_timetables( FM_OPL *OPL , int ARRATE , int DRRATE ) {
 	double rate;
 
 	/* make attack rate & decay rate tables */
-	for ( i = 0; i < 4; i++ ) OPL->AR_TABLE[i] = OPL->DR_TABLE[i] = 0; {
```

Comment:
Had to read this about 5 times to figure out what was happening in the original.  Thanks.